README.md

```diff
@@ -1,2 +1,113 @@
-# capstone-One
```

Comment:
- [ ] Kindly add the logo at the beginning of the readme template

Reply:
please the paragraph is there for the animation to work